articles/active-directory-b2c/openid-connect.md

```diff
@@ -31,31 +31,31 @@ Azure AD B2C extends the standard OpenID Connect protocol to do more than simple
 
 ```http
 GET https://{tenant}.b2clogin.com/{tenant}.onmicrosoft.com/{policy}/oauth2/v2.0/authorize?
 client_id=90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6
-&response_type=code+id_token
-&redirect_uri=https%3A%2F%2Faadb2cplayground.azurewebsites.net%2F
+&response_type=code+token
+&redirect_uri=https%3A%2F%2Fjwt.ms%2F
 &response_mode=fragment
-&scope=openid%20offline_access
+&scope=openid%20offline_access%20{application-id-uri}/{scope-name}
 &state=arbitrary_data_you_can_receive_in_the_response
 &nonce=12345
 ```
 
 | Parameter | Required | Description |
 | --------- | -------- | ----------- |
-| {tenant} | Yes | Name of your Azure AD B2C tenant |
+| {tenant} | Yes | Name of your [Azure AD B2C tenant](tenant-management.md#get-your-tenant-name). If you're using a [custom domain](custom-domain.md), replace `tenant.b2clogin.com` with your domain, such as `fabrikam.com`.  |
 | {policy} | Yes | The user flow to be run. Specify the name of a user flow you've created in your Azure AD B2C tenant. For example: `b2c_1_sign_in`, `b2c_1_sign_up`, or `b2c_1_edit_profile`. |
 | client_id | Yes | The application ID that the [Azure portal](https://portal.azure.com/) assigned to your application. |
 | nonce | Yes | A value included in the request (generated by the application) that is included in the resulting ID token as a claim. The application can then verify this value to mitigate token replay attacks. The value is typically a randomized unique string that can be used to identify the origin of the request. |
-| response_type | Yes | Must include an ID token for OpenID Connect. If your web application also needs tokens for calling a web API, you can use `code+id_token`. |
-| scope | Yes | A space-separated list of scopes. The `openid` scope indicates a permission to sign in the user and get data about the user in the form of ID tokens. The `offline_access` scope is optional for web applications. It indicates that your application will need a *refresh token* for extended access to resources. The `https://{tenant-name}/{app-id-uri}/{scope}` indicates a permission to protected resources, such as a web API. For more information, see [Request an access token](access-tokens.md#scopes). |
+| response_type | Yes | The response type, which must include `code` for the authorization code flow. If your web application also needs tokens for calling a web API, you can use `code+token`. When adding `token` to the response_type parameter value you must add a valid resource in the `scope` parameter. |
```

Comment:
```suggestion
| response_type | Yes | Must include an ID token for OpenID Connect. If your web application also needs tokens for calling a web API, you can use `code+id_token`. |
```

Reply:
retain original content!

Reply:
As mentioned earlier, I understand why we better retain the original content.
We are not changing anything that does not need to be changed!